[{"Date":"\/Date(1456992730480)\/","StackTrace":"   at FoodApp.Common.Managers.OrderManager.Buy(ngUserModel user, Int32 day, String foodId, Decimal val) in d:\\Projects\\git\\gam-gam\\WebSite\\Common\\Managers\\OrderManager.cs:line 37\r\n   at FoodApp.Controllers.api.FoodsController.Buy(String userId, Int32 day, String foodId, Decimal val) in d:\\Projects\\git\\gam-gam\\WebSite\\Controllers\\api\\FoodsController.cs:line 33\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.\u003c\u003ec__DisplayClass10.\u003cGetExecutor\u003eb__9(Object instance, Object[] methodParameters)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.\u003cInvokeActionAsyncCore\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ActionFilterResult.\u003cExecuteAsync\u003ed__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.\u003cSendAsync\u003ed__1.MoveNext()","Url":"http://localhost:15845/api/foods/47258346-5281-4fd7-9234-d5eb994c534c/3/_PershistravyCHanaxy/1/?time=1456992728214","ID":"59805b2e-c84b-4d9f-83ff-808e520110bd","UserName":"Міша Г","Error":"Object reference not set to an instance of an object."},{"Date":"\/Date(1456992730492)\/","StackTrace":"   at FoodApp.Common.Managers.OrderManager.Buy(ngUserModel user, Int32 day, String foodId, Decimal val) in d:\\Projects\\git\\gam-gam\\WebSite\\Common\\Managers\\OrderManager.cs:line 37\r\n   at FoodApp.Controllers.api.FoodsController.Buy(String userId, Int32 day, String foodId, Decimal val) in d:\\Projects\\git\\gam-gam\\WebSite\\Controllers\\api\\FoodsController.cs:line 33\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.\u003c\u003ec__DisplayClass10.\u003cGetExecutor\u003eb__9(Object instance, Object[] methodParameters)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.\u003cInvokeActionAsyncCore\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ActionFilterResult.\u003cExecuteAsync\u003ed__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.\u003cSendAsync\u003ed__1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.\u003cSendAsync\u003ed__1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Cors.CorsMessageHandler.\u003cSendAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.HttpServer.\u003cSendAsync\u003ed__0.MoveNext()","Url":"http://localhost:15845/api/foods/47258346-5281-4fd7-9234-d5eb994c534c/3/_PershistravyCHanaxy/1/?time=1456992728214","ID":"19269245-afa6-45c8-a59f-061d3437b471","UserName":"Міша Г","Error":"Object reference not set to an instance of an object."}]